cluster/local/integration_tests.sh

```diff
@@ -135,6 +145,14 @@ for ((i = 1; i <= 5; i++)); do
             echo_step_completed
         fi
 
+        echo_info "check if has error in logs"
+        error_logs=$("${KUBECTL}" -n "${CROSSPLANE_NAMESPACE}" logs --all-containers=true --timestamps=true --tail=10 "${name}" | grep -w ERROR || true)
```

Comment:
1. the script has no `pipefail` [option](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#set--o-pipefail), so result of whole command could be unpredictable.

2. you put as `${name}` the name of deployment and didn't use `deployment/` prefix, read here examples:
https://kubernetes.io/docs/reference/kubectl/cheatsheet/#interacting-with-deployments-and-services
current result:
    ```bash
    root@build-01fa6cad-inttests-control-plane:/# kubectl -n crossplane-system get pods                                                         
    NAME                                       READY   STATUS    RESTARTS   AGE
    crossplane-d44d85996-8ktkm                 1/1     Running   0          3m26s
    crossplane-d44d85996-wsc5c                 1/1     Running   0          3m26s
    crossplane-rbac-manager-6d4d74cdf6-9hdmq   1/1     Running   0          3m26s
    crossplane-rbac-manager-6d4d74cdf6-wxftk   1/1     Running   0          3m26s

    root@build-01fa6cad-inttests-control-plane:/# kubectl -n crossplane-system logs crossplane --all-containers=true --timestamps=true --tail=10
    Error from server (NotFound): pods "crossplane" not found
    ```

3. Why you use `--tail=10` ? What if errors not in last 10 lines?